### ISSUE TITLE
Color of composite doesn't equate the color of the most critical metric that compose it

### ISSUE THREAD

**Issue by matletix** [grafana/grafana-polystat-panel#158]
I did a simple test with two queries (aliases A and B) returning one point each (A with the value 2, B with the value 0).

I created 3 thresholds:
  - 0 for OK (green)
  - 1 for WARNING (yellow)
  - 2 for CRITICAL (red)

This way I have two hexagons, one for A showing red with a value of 2, the other for B showing green with a value of 0.
Then I created a composite with the 2 metrics A and B. If I put first the metric A which has a "critical" value, the composite shows RED as expected :

![grafana3](https://user-images.githubusercontent.com/19894690/98136638-7081ac80-1ec1-11eb-8a2d-29e08ed390bf.png)

But if I put B which has a "OK" value, as the first metric of the composite, it shows GREEN: 

![grafana](https://user-images.githubusercontent.com/19894690/98136889-af176700-1ec1-11eb-92ca-a91ad1b0bce5.png)

When hovering over the composite, the tooltip effectively shows the value of A and B:

![grafana2](https://user-images.githubusercontent.com/19894690/98137108-eab23100-1ec1-11eb-9aca-4e45c51f4181.png)

It seems that the color of the composite matches the color of its first metric, while I think it's expected that the color of the composite be the color of the highest threshold metric that compose it (red, for critical, in this case).

I tried with an elasticsearch data source and with an influxdb data source, both reproducing the issue.
Can someone else confirm this behaviour ? Is this expected behaviour or is it a bug ?

I ran grafana in a docker container with the following command: 
```bash
docker run -d \
-p 3000:3000 \
--name=grafana \
-e "GF_INSTALL_PLUGINS=grafana-polystat-panel" \
grafana/grafana
```

Grafana version : 7.2.1
grafana-polystat-panel version : 1.2.2

(also tried with grafana 6.7.3 and grafana-polystat-panel 1.2.1 with no success)

**Comment by matletix** [comment]
Update : creating one override for each metric that is part of the composite, with the same thresholds than the metrics, make the composite color behave as expected (color of the most critical metric).

**Comment by MAzZYBig** [comment]
Confirm that bug.
Very annoying

**Comment by matletix** [comment]
Fixed by #174

**Comment by briangann** [comment]
great catch, i've normally configured overrides and hadn't seen this.
